### ISSUE TITLE
Example of increasing dropout rate throughout training

### ISSUE THREAD

**Issue by LukeWood** [keras-team/keras-io#839]
This can go over the solution of:
https://github.com/keras-team/keras/issues/16234
